decode.go

```diff
@@ -396,3 +398,18 @@ func floatSetter(bitSize int) valueSetter {
 		return nil
 	}
 }
+
+func boolSetter(v reflect.Value, raw rawValue) error {
+	if len(raw.data) == 0 {
+		return nil
+	}
+
+	trimmedBool := strings.TrimSpace(raw.data)
+	val, err := strconv.ParseBool(trimmedBool)
```

Comment:
I like the use of `strconv.ParseBool` here, but some documentation should be added specifying what the acceptable values are.

> // ParseBool returns the boolean value represented by the string.
// It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False.
// Any other value returns an error.

Also, currently, an empty string will result in an error. I'm not sure this is the desired behavior. I'd expect the result to simply be `false` (I definitely don't feel strongly about this, happy to hear any thoughts you may have).

Reply:
@ianlopshire added in readme example for short bool usage (f, F, t, T)

> Also, currently, an empty string will result in an error.

I wrote tests for case when we have empty line - works fine now.
The only thing I didn’t added (tests) but that will work is for rest of cases like 1, 0, TRUE, FALSE etc. And I believe another questions is either we need to encode bool struct field in different way (f instead of false, or 1 instead of true), but that is a discussion for different pr I guess which requires documentation.